flexget/plugins/clients/pyload.py

```diff
@@ -155,11 +155,17 @@ def add_entries(self, task, config):
 
         for entry in task.accepted:
             # bunch of urls now going to check
-            content = entry.get('description', '') + ' ' + quote(entry['url'])
-            content = json.dumps(content)
+            contents = []
+            description = entry.get('description', '')
+            if description != '':
```

Comment:
Generally the pythonic way is to rely on the None value returned if key is not set

ie.

```
desc = entry.get('description')
if desc is not None:
  contents.append(desc)
```
Unless you specifically want to exclude empty description